lib/src/test/resources/http4s/apidoc-api/018/BryzekApidocApiV0Client.scala.txt

```diff
@@ -3863,6 +3850,21 @@ formBody.fold(Sync[F].pure(reqAndMaybeAuth))(reqAndMaybeAuth.withBody)
 
   package errors {
 
+    import io.apibuilder.api.v0.models.json._
+    import io.apibuilder.common.v0.models.json._
+    import io.apibuilder.generator.v0.models.json._
+    import io.apibuilder.spec.v0.models.json._
```

Comment:
Are these needed?

Reply:
Thanks @gheine. Good point. They are not. While those imports are for Play, and http4s `0.15`, `0.17`, they are not needed here.

Updated code to remove those json imports. Then tested proposed changes locally with http4s `0.18`, and `0.20` (with manual application of changes -- as I don't have apibuilder running locally -- btw. is there any `docker-compose.yml` for that?!).

Should be good now. Please take a look again @gheine.